src/helpers/cards.ts

```diff
@@ -321,6 +323,165 @@ export const changeCardStatus = async (
   return cardData.card;
 };
 
+/**
+ * Triggers a series of webhooks calls (CARD_TOKEN_LIFECYCLE) simulating the provisioning token creation
+ * lifecycle. When done, it saves the final token in the card information of the person.
+ *
+ * @param personId {string}
+ * @param cardId {string}
+ */
+export const createProvisioningToken = async (
+  personId: string,
+  cardId: string,
+): Promise<ProvisioningTokenStatusChangePayload> => {
+
+  if (!personId) {
+    throw new Error("You have to provide personId");
+  }
+  if (!cardId) {
+    throw new Error("You have to provide cardId");
+  }
+
+  const person = await db.getPerson(personId);
+  const cardData = person.account.cards.find(({ card }) => card.id === cardId);
+  if (!cardData) {
+    throw new Error("Card not found");
+  }
+
+  const { provisioningToken } = cardData;
+  let walletId;
+
+  // Deactivate existing token
+  if (provisioningToken) {
+    walletId = provisioningToken.client_wallet_account_id
```

Comment:
nit: missing trailing `;`

Reply:
I added pretty-quick [here](https://github.com/kontist/mock-solaris/pull/8/files) which should make it easier to take care of all these nits.